TurkishFilter/sections/specific.txt

```diff
@@ -440,6 +440,7 @@ fullhdfilmcibaba.com##img[src*=".com/rek/"]
 filmilla.com###sayfa a[href][target="_blank"] > img
 filmilla.com##body > #sayfa > div[style="width:100%;text-align: center;margin-bottom: 12px;"]
 /(dikey|horay)\.jpg/$domain=pusuhaber.net
+||siyahfilmizle.com/wp-content/uploads/2021/06/redfoxbet-pageskin.jpg
```

Comment:
`||siyahfilmizle.com/wp-content/uploads/*/redfoxbet-pageskin.jpg`.
Missed ad here https://siyahfilmizle.com/the-many-saints-of-newark-2021-4k-izle/
<details><summary>Screenshot:</summary>

![image](https://user-images.githubusercontent.com/33488229/124093043-2658c700-da60-11eb-9221-1b420843990a.png)

</details><br/>